c/tests/test_core.c

```diff
@@ -526,6 +526,17 @@ test_avl_random(void)
     validate_avl(sizeof(keys) / sizeof(*keys), keys);
 }
 
+static void
+test_tskit_version(void)
```

Comment:
We're testing the meson project version agrees with the defines (which are authoritative). Good to name the test accordingly